crates/gpui/src/platform/linux/text_system.rs

```diff
@@ -74,9 +75,14 @@ impl PlatformTextSystem for LinuxTextSystem {
             .collect()
     }
 
-    // todo!(linux)
     fn all_font_families(&self) -> Vec<String> {
-        Vec::new()
+        self.0
+            .read()
+            .font_system
+            .db()
+            .faces()
+            .map(|face| face.families.clone()[0].clone().0)
```

Comment:
Could this be a `.filter_map()` + a `.get(0)`? Also, we shouldn't have to clone the whole vec to do this.

Reply:
I've made the recommended changes. Please let me know if any more changes are required.